### PR TITLE
Publish documentation updates for organizations

### PR DESCRIPTION
**Pull request by tpdorsey**
Publish updates to Organization docs, including roles from https://github.com/PrefectHQ/prefect/pull/7058

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: docs/ui/organizations.md [hunk 1]
```diff
@@ -12,17 +12,85 @@ tags:
...
+Within a workspace, select **Workspace Sharing**, then select the **+** icon to add new members or service accounts to the workspace. Only organization Admins and workspace Owners may add members or service accounts to a workspace.
+
+![Organization workspace sharing in Prefect Cloud.](/img/ui/org-workspace-sharing.png)
+
+Members and service accounts must already be configured for the organization. An Admin or Owner may configure a different role for the user or service account as needed.
+
+!!! note "Default workspace role"
+    You may make a workspace available to anyone in an organization by settings a default role for "Anyone at...". Anyone in the organization may access the workspace with the specified default role permissions.
+
+    The role given to users specifically added to the workspace overrides this setting.
```
Comment (zangell44) [inline docs/ui/organizations.md:79]: It doesn't quite override it, it should give any user the union of workspace scopes given by the "Default workspace role" and that users' role in the workspace

File: docs/ui/organizations.md [hunk 2]
```diff
@@ -12,17 +12,85 @@ tags:
...
+
+You may give organization members and service accounts access to workspaces within an organization. Each workspace within an organization may have its own members and service accounts with roles and permissions specific to that workspace. Organization Admins have full access to all workspaces in an organization.
+
+Within a workspace, select **Workspace Sharing**, then select the **+** icon to add new members or service accounts to the workspace. Only organization Admins and workspace Owners may add members or service accounts to a workspace.
+
+![Organization workspace sharing in Prefect Cloud.](/img/ui/org-workspace-sharing.png)
+
+Members and service accounts must already be configured for the organization. An Admin or Owner may configure a different role for the user or service account as needed.
+
+!!! note "Default workspace role"
```
Comment (zangell44) [inline docs/ui/organizations.md:76]: also probably worth noting the default role does not apply to service accounts, only users

**Review by zangell44 — APPROVED** [review]

File: docs/ui/organizations.md [hunk 3]
```diff
@@ -12,17 +12,85 @@ tags:
 
 # Organizations in Prefect Cloud
 
-For larger teams, and companies with more complex needs around user and access management, annual plans include organizations in Prefect Cloud 2. Organizations provide a host of ways to collaborate securely at scale, including:
+For larger teams, and companies with more complex needs around user and access management, organizations provide several features that enable you to collaborate securely at scale, including:
```
Comment (billpalombi) [inline docs/ui/organizations.md:15]:
```suggestion
For larger teams or companies with more complex needs around user and access management, organizations provide several features that enable you to collaborate securely at scale, including:
```

File: docs/ui/organizations.md [hunk 4]
```diff
@@ -12,17 +12,85 @@ tags:
...
+
+**Workspaces** shows you a list of workspaces within the organziation to which you have access.
+
+## Organization members
+
+**Members** shows you a list of users who are members of the organization. If you have been given the organization Admin role, you can invite new members and set organization roles for users here.
+
+![Viewing the Organization member page in Prefect Cloud.](/img/ui/org-members.png)
+
+Control granular access to workspaces by setting default access for all organization members, inviting specific members to collaborate in an organization workspace, or adding service account permissions.
```
Comment (billpalombi) [inline docs/ui/organizations.md:39]:
```suggestion
You can control granular access to workspaces by setting default access for all organization members, inviting specific members to collaborate in an organization workspace, or adding service account permissions.
```

File: docs/ui/organizations.md [hunk 5]
```diff
@@ -12,17 +12,85 @@ tags:
...
+!!! note "Default workspace role"
+    You may make a workspace available to anyone in an organization by settings a default role for "Anyone at...". Anyone in the organization may access the workspace with the specified default role permissions.
+
+    The role given to users specifically added to the workspace overrides this setting.
+    
+    You may set this to No Access if you do not want organization members not specifically added to the workspace to access it (organization Admins excepted). 
+
+## Organization and workspace roles
+
+Prefect Cloud enables configuring both organization and workspace roles for users.
```
Comment (billpalombi) [inline docs/ui/organizations.md:85]:
```suggestion
Prefect Cloud enables configuration of both organization and workspace roles for users.
```

**Review by billpalombi — APPROVED** [review]

**Pull request merged**